src/components/ItemDetail/ReturnDialog.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <dialog-template width="480px" @close="close">
+    <h2 :class="$style.title">物品を返却する</h2>
+    <form :class="$style.container" @submit="returnItemAndClose">
```

Comment:
```suggestion
    <form :class="$style.container" @submit.prevent="returnItemAndClose">
```
`borrow`のほうで言い忘れてたけどこれ必要だね